less/variables.less

```diff
@@ -11,4 +11,15 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+
+// ## Task 5: Mobile Updates Needed
+// * [ ] Use escaping to create a variable named 
+// `@mobile` that contains this value: `(max-width: 500px)`. 
+//  Use the `variables.less` file to house your variables.
+
+// excaping variables
+
+
+@mobile: ~"(max-width: 500px)";
```

Comment:
Although this @mobile is set to 500px it appears your code has 2 breakpoints. Your flexbox property might be wrapping by default.
![image](https://user-images.githubusercontent.com/5153613/66745195-6ee8df00-ee8f-11e9-96d7-3a651b6614f8.png)
![image](https://user-images.githubusercontent.com/5153613/66745230-80ca8200-ee8f-11e9-87b6-0ecf20998cbf.png)